{{cookiecutter.github_repository}}/provisioner/vars.yml

```diff
@@ -5,3 +5,4 @@ project_repo_remote: origin
 repo_version: master
 pg_gis: {{ 'True' if cookiecutter.postgis.lower() == 'y' else 'False' }}
 sass_with_django_compressor: {{ 'True' if cookiecutter.add_sass_with_django_compressor.lower() == 'y' else 'False' }}
+enable_whitenoise: {{ 'True' if cookiecutter.enable_whitenoise.lower() == 'y' else 'False'}}
```

Comment:
Let's not hv it as a variable here. In the nginx config, better use the `cookiecutter.enable_whitenoise` have it as one time thing.